mlflow/protos/service.proto

```diff
@@ -453,6 +467,14 @@ enum RunStatus {
   KILLED = 5;
 }
 
+// Types of vertices represented in MLflow Run Inputs. Valid vertices are MLflow objects that can
+// have an input relationship.
+enum InputVertexType {
+  RUN = 1;
+
+  DATASET = 2;
+}
```

Comment:
I'd prefer to keep this internal to Databricks, if possible. I don't think it needs to be part of the MLflow public REST API